src/api.js

```diff
@@ -1,13 +1,29 @@
 import axios from "axios";
 
+const ncGamesAPI = axios.create({
+    baseURL: 'https://nc-jpportfolio.onrender.com/api'
+});
+
 export const fetchReviews = () => {
-  return axios
-    .get(`https://nc-jpportfolio.onrender.com/api/reviews`)
+  return ncGamesAPI
+    .get(`/reviews`)
     .then((response) => {
       return response.data.reviews;
     })
     .catch((error) => {
       console.error("Error fetching reviews:", error);
       throw error;
     });
-};
\ No newline at end of file
+};
+
+export const fetchReview = (review_id) => {
+    return ncGamesAPI
+      .get(`/reviews/${review_id}`)
+      .then((response) => {
+        return response.data.review;
+      })
+      .catch((error) => {
+        console.error(`Error fetching review with that Id ${review_id}:`, error);
+        throw error;
+      });
```

Comment:
Dont have the catch here - should have the single catch when you invoke the function in the frontend.